src/Integration/Telemetry/ShortIsoDateTimeOffsetConverter.cs

```diff
@@ -34,7 +34,17 @@ internal class ShortIsoDateTimeOffsetConverter : JsonConverter
         public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer) =>
             reader.ReadAsDateTimeOffset();
 
-        public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer) =>
-            writer.WriteValue(((DateTimeOffset)value).ToString("yyyy-MM-ddTHH:mm:ss.fffzzz"));
+        public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
+        {
+            // Use the standard format string for ISO 8601
+            var iso8601Date = ((DateTimeOffset)value).ToString("o");
+
+            // Remove the ticks part (not expected by our server side)
+            const int millisecondLastIndex = 23;
+            const int timeOffsetFirstIndex = 27;
+            var simplifiedDate = iso8601Date.Substring(0, millisecondLastIndex) + iso8601Date.Substring(timeOffsetFirstIndex);
```

Comment:
Perhaps write a test ensuring that the width of the date parts is always the same, e.g. with single digit month, day, hour, min, sec, etc.